packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -589,8 +589,12 @@ export const configureVaultFactoryUI = async ({
   const boardIdValue = [
     ['INSTANCE_BOARD_ID', instances.vaultFactory],
     ['INSTALLATION_BOARD_ID', installs.vaultFactory],
-    ['RUN_ISSUER_BOARD_ID', centralIssuer],
-    ['RUN_BRAND_BOARD_ID', centralBrand],
+    // @deprecated
+    ['RUN_ISSUER_BOARD_ID', stableIssuer],
+    // @deprecated
+    ['RUN_BRAND_BOARD_ID', stableBrand],
+    ['STABLE_ISSUER_BOARD_ID', stableIssuer],
+    ['STABLE_BRAND_BOARD_ID', stableBrand],
```

Comment:
things like this really should be looked up from a namehub such as `E(home).agoricNames` (see also #4819), so adding features to `configureVaultFactoryUI` seems like the wrong direction. Not critical, I suppose.

Reply:
Yeah. Easy enough to remove when that work is done. Meanwhile I don't want to slow down the dapps moving off RUN_*